### DISCUSSION TITLE
单机部署，显示部署成功。但是没有监听443端口

### DISCUSSION THREAD

**Discussion by caisirius** [labring/sealos#4236] [Q&A]
手动下载了 sealos_4.3.5_linux_amd64.tar.gz，解压后拷贝到 /usr/local/bin。
再执行 install.sh
显示是成功了。
但是发现 443端口未被监听。不知道怎么去访问

**Comment by bxy4543** [comment]
你的这个只是成功安装二进制的 并非成功安装desktop，官网文档有一键安装命令 可以看下

**Comment by caisirius** [comment]
@bxy4543 
install.sh 是整体安装的脚本
安装完了后：
<img width="859" alt="image" src="https://github.com/labring/sealos/assets/6105307/6dae73d3-2522-4703-a7af-1d161a073084">

应该是有desktop 运行起来了的。
但是 没有443端口被监听。
<img width="781" alt="image" src="https://github.com/labring/sealos/assets/6105307/a40b29a5-d9f6-4cf4-9cba-25fd8b531218">

**Comment by bxy4543** [comment]
@caisirius 你install.sh脚本是哪个脚本呢？
https://sealos.io/docs/self-hosting/sealos/installation 这个文档里的吗？，执行过程是什么

**Reply by caisirius** [reply]
因为网络原因，我是10.30下载的 https://raw.githubusercontent.com/labring/sealos/main/scripts/cloud/install.sh，到我电脑再上传到服务器的 install.sh
 sealos_4.3.5_linux_amd64.tar.gz也是手动下载再解压的

**Reply by caisirius** [reply]
执行过程已经看不到了。要不我再重新下载一次 install.sh 再去执行一次？

**Reply by caisirius** [reply]
记得执行过程是这样：
cd /usr/local/bin
wget ..../sealos_4.3.5_linux_amd64.tar.gz
tar xzf sealos_4.3.5_linux_amd64.tar.gz

cd ~
wget  ..../install.sh
bash install.sh


然后在交互界面里输入的域名是   10.194.49.154.nip.io
端口没有输入，默认443
其他都是默认

**Reply by bxy4543** [reply]
kubectl get pod -A 看下

**Reply by caisirius** [reply]
刚刚重新安装了，报错了：
<img width="1419" alt="image" src="https://github.com/labring/sealos/assets/6105307/1a1217e9-f95f-4283-a715-57b62201f815">

然后：
<img width="997" alt="image" src="https://github.com/labring/sealos/assets/6105307/2e3efa4d-3d5f-4f4b-bfe7-92572edcff35">

**Reply by bxy4543** [reply]
镜像有点问题，会更新一个修复版

**Reply by caisirius** [reply]
更新好了麻烦说下
